pkg/metadata/gvk.go

```diff
@@ -0,0 +1,33 @@
+package metadata
+
+import (
+	"github.com/opendatahub-io/odh-platform/pkg/spi"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+func ExternalGVKs() []schema.GroupVersionKind {
+	return []schema.GroupVersionKind{
+		{Group: "route.openshift.io", Version: "v1", Kind: "Route"},
+		{Group: "networking.istio.io", Version: "v1beta1", Kind: "VirtualService"},
+	}
+}
+
+func PublicGVKs() []schema.GroupVersionKind {
+	return []schema.GroupVersionKind{
+		{Group: "", Version: "v1", Kind: "Service"},
+		{Group: "networking.istio.io", Version: "v1beta1", Kind: "Gateway"},
+		{Group: "networking.istio.io", Version: "v1beta1", Kind: "VirtualService"},
+		{Group: "networking.istio.io", Version: "v1beta1", Kind: "DestinationRule"},
+	}
+}
```

Comment:
These are static values, no need to wrap it in `funcs` I believe.

Reply:
[72a9617](https://github.com/maistra/odh-platform/pull/39/commits/72a961743f9f3b6976e1dbc9a590e274185da5f0)